### PR TITLE
Add pipeline cleanup on termination

### PR DESCRIPTION
**Pull request by gyng**
* Stop in-flight pipelines on cleanup
  - Minimises potential bad task runs, especially if workloads have side-effects and are not independent
* Add `.travis.yml`, black, flake8
* Add dev airflow docker-compose

Note that the operator and example DAG are hardlinked to the `dev/docker` directory, causing the diff linecount to be inaccurate. Not ideal, but acceptable for now.

While adding CI I noticed that our operator doesn't work on Python 2.7 and 3.5. An issue has been filed for 3.5: #10

### CODE REVIEW

File: bedrock_plugin.py [hunk 1]
```diff
@@ -30,7 +30,10 @@ def get_conn(self, _headers=None):
 
         session = requests.Session()
         session.headers.update(
-            {"Authorization": f"Bearer {self.bedrock_token}", "Content-Type": "application/json"}
+            {
+                "Authorization": f"Bearer {self.bedrock_token}",
```
Comment (chrissng) [inline bedrock_plugin.py]: I'll hesitate using f-strings as Airflow is supported in Python versions 3.5 and lower. For maximizing compatibility, I would suggest str.format()

File: dev/docker/plugins/bedrock_plugin.hardlink.py [hunk 2]
```diff
@@ -0,0 +1,213 @@
+#!/usr/bin/python3
```
Comment (chrissng) [inline dev/docker/plugins/bedrock_plugin.hardlink.py]: Can we simply symlink to the actual python script?

File: bedrock_plugin.py (hunk 1 above)
Comment (gyng) [reply bedrock_plugin.py]: See #10 for fixing this. This diff is just due to applying black on the code.

File: dev/docker/plugins/bedrock_plugin.hardlink.py (hunk 2 above)
Comment (gyng) [reply dev/docker/plugins/bedrock_plugin.hardlink.py]: It didn't seem to work in the docker image with a relative softlink.

File: bedrock_plugin.py (hunk 1 above)
Comment (chrissng) [reply bedrock_plugin.py]: Okay sure

File: dev/docker/plugins/bedrock_plugin.hardlink.py (hunk 2 above)
Comment (gyng) [reply dev/docker/plugins/bedrock_plugin.hardlink.py]: Removed the hardlinks. Just get devs to symlink manually for dev.

**Pull request merged**